src/test/java/com/hubspot/jinjava/interpret/LazyExpressionTest.java

```diff
@@ -29,4 +32,31 @@ public void itSerializesNonEvaluatedValueToEmpty() throws JsonProcessingExceptio
     );
     assertThat(new ObjectMapper().writeValueAsString(expression)).isEqualTo("\"\"");
   }
+
+  @Test
+  public void itMemoizesByDefault() {
+    List mock = mock(List.class);
+    LazyExpression expression = LazyExpression.of(mock::isEmpty, "");
+    expression.get();
+    expression.get();
+    verify(mock, times(1)).isEmpty();
```

Comment:
```suggestion
    verify(mock).isEmpty();
```

`times(1)` is the default